<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="23076" started-at="2020-01-07T23:08:16 GMT" finished-at="2020-01-07T23:08:39 GMT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="23076" started-at="2020-01-07T23:08:16 GMT" finished-at="2020-01-07T23:08:39 GMT">
      <class name="com.vtiger.testcases.VtigerSmokeTestCases">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33]" name="setup" is-config="true" duration-ms="148" started-at="2020-01-07T23:08:16 GMT" finished-at="2020-01-07T23:08:16 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="applicationSetup()[pri:0, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33]" name="applicationSetup" is-config="true" duration-ms="11000" started-at="2020-01-07T23:08:16 GMT" finished-at="2020-01-07T23:08:27 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- applicationSetup -->
        <test-method status="FAIL" signature="createLead()[pri:1, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33]" name="createLead" duration-ms="32" started-at="2020-01-07T23:08:27 GMT" finished-at="2020-01-07T23:08:27 GMT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Not able to input the value did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Not able to input the value did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at utilities.GenericMethod.setTextboxValue(GenericMethod.java:96)
at pages.Login.loginToApplication(Login.java:13)
at com.vtiger.testcases.VtigerSmokeTestCases.createLead(VtigerSmokeTestCases.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createLead -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33]" name="tearDownMethod" is-config="true" duration-ms="946" started-at="2020-01-07T23:08:27 GMT" finished-at="2020-01-07T23:08:28 GMT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=createLead status=FAILURE method=VtigerSmokeTestCases.createLead()[pri:1, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="applicationSetup()[pri:0, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33]" name="applicationSetup" is-config="true" duration-ms="10157" started-at="2020-01-07T23:08:28 GMT" finished-at="2020-01-07T23:08:39 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- applicationSetup -->
        <test-method status="FAIL" signature="leadData()[pri:2, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33]" name="leadData" duration-ms="19" started-at="2020-01-07T23:08:39 GMT" finished-at="2020-01-07T23:08:39 GMT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Not able to input the value did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Not able to input the value did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at utilities.GenericMethod.setTextboxValue(GenericMethod.java:96)
at pages.Login.loginToApplication(Login.java:13)
at com.vtiger.testcases.VtigerSmokeTestCases.leadData(VtigerSmokeTestCases.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- leadData -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33]" name="tearDownMethod" is-config="true" duration-ms="898" started-at="2020-01-07T23:08:39 GMT" finished-at="2020-01-07T23:08:39 GMT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=leadData status=FAILURE method=VtigerSmokeTestCases.leadData()[pri:2, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="finalReport()[pri:0, instance:com.vtiger.testcases.VtigerSmokeTestCases@167fdd33]" name="finalReport" is-config="true" duration-ms="232" started-at="2020-01-07T23:08:40 GMT" finished-at="2020-01-07T23:08:40 GMT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finalReport -->
      </class> <!-- com.vtiger.testcases.VtigerSmokeTestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
